content/components/minimize-logic-in-templates.md

```diff
@@ -39,3 +39,7 @@ export class SomeComponent {
   }
 }
 ```
+
+### Gotchas
```

Comment:
I would consider renaming it to 'Best practices' as 'Gotchas' might confuse non-native English speakers not being used to seeing such a term used as a title (Gotchas having different meanings).

Reply:
`Best Practices` sounds good to me 👍